cluster/ubuntu/util.sh

```diff
@@ -188,8 +188,8 @@ function create-etcd-opts() {
   cat <<EOF > ~/kube/default/etcd
 ETCD_OPTS="\
  -name infra\
- -listen-client-urls http://0.0.0.0:4001\
- -advertise-client-urls http://127.0.0.1:4001"
+ -listen-client-urls http://127.0.0.1:4001,http://${1}:4001\
```

Comment:
> Not sure why you want to listen on localhost and also advertise localhost.

Why should we avoid this? Does it apply to all situations? 
Say i wanna to bring up a cluster, all locating in a single (and the same) machine (kinda odd, i'll give you that), isn't it suppose to advertise localhost(different port though)? 
Thanks ahead:)